hw2/04-background.html

```diff
@@ -6,29 +6,39 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
-  <body class = "background">
+  <div id="background">
+  <body>
+  </div>
     <main>
       <div class="container my-5">
-      <div class="card bg-light mb-5 mx-auto"
-      style="max-width: 30rem; height: 18rem">
-      <h1 class="my-3 px-2"><strong>Exercise 04 - Background</strong></h1>
-      <p class="px-2">
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label class="px-2">Background change interval (seconds): </label>
-        <input class="mx-2 mt-1" type="number" id="number"/>
-      </div>
-      <div>
-        <input class="btn btn-primary my-3 mx-2" type="submit" value="Start" id="submit">
+        <div
+          class="card bg-light mb-5 mx-auto"
+          style="max-width: 30rem; height: 18rem"
+        >
+          <h3 class="my-3 px-3"><strong>Exercise 04 - Background</strong></h1>
+          <p class=" px-3">
+            The background color will change every few seconds (as indicated
+            below). You can stop this behavior and you can also change how often
+            the background color changes.
+          </p>
+          <div>
+            <label class="px-3">Background change interval (seconds): </label>
```

Comment:
You are having the same accessibility as the previous program, for attribute to label will help you get rid of this!

Reply:
thanks